compiler/src/ammtoaga/depgraph.ts

```diff
@@ -540,6 +608,6 @@ export const opcodeParamMutabilities = {
   seqeach: [false, null],
   seqwhile: [false, null, null], // TODO: ok so i don't *want* to make the 2nd value `null`, but it's not impossible for someone to mutate a value in the second function...
   seqdo: [false, null],
-  selfrec: [null, null], // TODO: figure this out. maybe just mark both as false??
+  selfrec: [false, false], // TODO: figure this out. maybe just mark both as false??
```

Comment:
Delete the TODO? Or replace it with a new TODO to tackle the recursive dependencies better?